polus-multichannel-tiff-plugin/Dockerfile

```diff
@@ -0,0 +1,18 @@
+
+FROM labshare/polus-bfio-util:1.3.6
```

Comment:
Use bfio version `2.0.4`  ?

Reply:
Actually, I'm going to leave this as is. The major advantage to using bfio v2 is for the Python support. There would be some advantage to moving this to bfio v2, but this is a bit of a one-off plugin created for a specific demo. We want to include it `polus-plugins` in case we ever need something like this again.